modules/java/pages/gs-prereqs.adoc

```diff
@@ -1,76 +1,83 @@
+To verify whether your device supports the AVX2 instructions set, https://www.intel.com/content/www/us/en/support/articles/000090473/processors/intel-core-processors.html[follow these instructions.]
 --
 
+[#introduction]
 == Introduction
 
-{cblJP} enables development and deployment of {cbl} applications to a JVM environment.
-You can deploy to a desktop or web server (for example, {gpWebServerEnv}), including embedded {gpWebServerEnv} deployments.
+Couchbase{nbsp}Lite enables development and deployment of Couchbase{nbsp}Lite applications to a JVM environment.
+You can deploy to a desktop or web server (for example, Tomcat), including embedded Tomcat deployments.
 
-NOTE: We use _{gpIDE}_, _{gpBuildTool}_, _Maven_ and _{gpWebServerEnv}_ as tools to build and run the examples presented in this content.
+NOTE: We use _Intellij IDEA_, _gradle_, _Maven_ and _Tomcat_ as tools to build and run the examples presented in this content.
 You are free to use the tools of your choice.
 
 
 [#bmkSetupSyncAndServer, Setting up Sync Gateway]
 
+[#bmkSetupSyncAndServer]
 == Install Server Software
 
-If you want to use {cbljp} with {svr} and-or {sg} you will need to have installed operational instances of these before completing the installation and test build steps.
+If you want to use Couchbase{nbsp}Lite with _Couchbase{nbsp}Server_ and-or _Sync{nbsp}Gateway you will need to have installed operational instances of these before completing the installation and test build steps.
 
-So, If you have not already done so, see {sgw-prepare--xref}
+So, If you have not already done so, see xref:sync-gateway::get-started-prepare.adoc[Prepare Sync Gateway]
 
-* Within {svr}, create a bucket named {nmSampleAppDb}
-* Create a RBAC user for {sg} with username = `sync-gateway` and password = `password`
+* Within _Couchbase{nbsp}Server_, create a bucket named getting-started.cblite2
+* Create a RBAC user for _Sync{nbsp}Gateway with username = `sync-gateway` and password = `password`
 
 
+[#macos-and-windows]
 == MacOS and Windows
 
-You may now proceed directly to xref:gs-install.adoc[].
+You may now proceed directly to xref:java:gs-install.adoc[].
 
 [#additional-steps-for-linux]
 == Additional Steps For Linux
 
-Before proceeding to xref:gs-install.adoc[], you will need to make the supplied support libraries available to your running application.
+Before proceeding to xref:java:gs-install.adoc[], you will need to make the supplied support libraries available to your running application.
 
+[#steps]
 === Steps
 
-. Download the _zip_ file from here -- https://packages.couchbase.com/releases/couchbase-lite-java/{version-maintenance-java}/couchbase-lite-java-linux-supportlibs-{version-maintenance-java}.zip
+. Download the _zip_ file from here -- https://packages.couchbase.com/releases/couchbase-lite-java/{major}.{minor}.{maintenance-java}{empty}/couchbase-lite-java-linux-supportlibs-{major}.{minor}.{maintenance-java}{empty}.zip
```

Comment:
Not sure, but it looks like `{empty}` isn't defined by us, and isn't even a builtin asciidoc/Antora attribute?
e.g. it resolves to the empty string, but only accidentally... 🙀 

I suspect it "does no harm" here, but probably worth deleting if it also does nothing useful.

Reply:
Ah yes, the pattern is used in the `====== {empty}` hack... 
So we can't just delete it globally, but could try something more precise if Simon thinks worth doing.